packages/gated-content/src/conditions/__tests__/nft-condition.spec.ts

```diff
@@ -137,14 +137,84 @@ describe(`Given the "${transformNftCondition.name}" function`, () => {
     },
 
     {
-      description: 'a NFT Ownership condition that has invalid contract type',
+      description: 'a ERC20 Ownership condition that has invalid contract type',
       condition: mockNftOwnershipInput({
         chainID: ethereumChainId,
         contractAddress: knownAddress,
         contractType: ContractType.Erc20,
```

Comment:
I might have wrote this but still I am confused by this. Why "NFT ownership" with `ContractType.Erc20`?

Reply:
Seems like a leftover as this test is also repeated in the `erc20-condition.spec.ts`, removed